### ISSUE TITLE
Script update

### ISSUE THREAD

**Issue by wolffe** [Mango/slideout#243]
Is this script going to be updated soon? I see lots of PR's but no update in a long time.

**Comment by redpik** [comment]
Same question. Did you find an alternative?

**Comment by wolffe** [comment]
No, I am using this one because I like it and I don't have any issues. I implemented it with 2 sites. When I'm done with the testing, I'll see if I can incorporate some of the pull requests here.
